src/octopeer-github/main/Settings/Settings.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Created by Mitchell on 12-5-2016.
+ */
+
+/**
+ * Class from which user settings can be retrieved.
+ * These settings have been maintained by the front-end.
+ */
+class Settings {
```

Comment:
Oké, yes. Could you please have a look at that @mdingjan? The changes are well done! This is the only thing in the PR bothering me